src/utils/api/channels/vrpc/requests/getVdxfid.js

```diff
@@ -0,0 +1,12 @@
+import VrpcProvider from "../../../../vrpc/vrpcInterface"
+
+export const getVdxfId = async (system_id, vdxfuri, initialdata) => {
```

Comment:
This would be better achieved through VrpcProvider.getEndpoint(system_id).getVdxfId, as opposed to getVerusIdInterface, then calling the interface.